tests/400-serlo-editor.ts

```diff
@@ -89,3 +89,290 @@ Scenario('Markdown list shortcut', async ({ I }) => {
 
   I.seeElement('ul[data-slate-node="element"]')
 })
+
+Scenario('Plugin Suggestions Menu', async ({ I }) => {
+  I.amOnPage('/entity/create/Article/1377')
+
+  //Upload first image
+
+  I.click("input[placeholder='https://example.com/image.png']")
+
+  I.type('https://denkspace.de/zukunftswerk.gif')
+
+  //Click on “Füge ein Element hinzu” to get the menu
+
+  I.click('Füge ein Element hinzu')
+
+  I.click('span[data-slate-node="text"]')
+
+  I.seeElement('div.sc-fbYMXx.dTXjUa') //→ plugin suggestion menu
```

Comment:
those identifiers (e.g. `sc-fbYMXx.dTXjUa`) are auto-generated by styled components so it's not a working way to find elements:
1. we will have removed all styled components soon 
2. they change and won't work reliably